bundles/rust/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2016 Alejandro Baez https://twitter.com/a_baez
```

Comment:
FWIW, normally there's "just" the [copyright stanza](https://github.com/howl-editor/howl/blob/master/bundles/pascal/init.moon#L1-L2) at the top of each file ([which I usually forget](https://github.com/howl-editor/howl/pull/181/commits/5c331574657a683ac28198b18a4ff3cc5a940376)).